CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Smithy Typescript Codegen Changelog
 
+## 0.29.1 (2025-04-24)
+
+### Features
+- Created SignatureV4a JavaScript implementation ([#1319](https://github.com/smithy-lang/smithy-typescript/pull/1319))
+
```

Comment:
This is not a codegen change, should we remove it?
```suggestion
```

Also, the https://github.com/aws/aws-sdk-js-v3/pull/6267 is not expected to be in the equivalent aws codegen release.